ajax/libs/vexflow/1.2.84/vexflow-tests.js

```diff
@@ -0,0 +1,13838 @@
+/**
+ * VexFlow 1.2.84 built on 2017-07-30.
+ * Copyright (c) 2010 Mohit Muthanna Cheppudira <mohit@muthanna.com>
+ *
+ * http://www.vexflow.com  http://github.com/0xfe/vexflow
+ *//**
+ * VexFlow Test Support Library
```

Comment:
@sufuf3 the comment say that this is vexflow test support library, so I think this can be included too?